flow/raster_cache.h

```diff
@@ -150,16 +150,14 @@ class RasterCache {
       bool checkerboard) const;
 
   static SkRect GetDeviceBounds(const SkRect& rect, const SkMatrix& ctm) {
-#ifndef SUPPORT_FRACTIONAL_TRANSLATION
     SkRect device_rect;
     ctm.mapRect(&device_rect, rect);
+#ifndef SUPPORT_FRACTIONAL_TRANSLATION
     SkIRect bounds;
     device_rect.roundOut(&bounds);
     return SkRect::MakeLTRB(bounds.fLeft, bounds.fTop, bounds.fRight,
                             bounds.fBottom);
```

Comment:
This code is equivalent to `device_rect.roundOut(&deviceRect); return device_rect;` which makes the return statements the same in both cases.

Reply:
Basically, I believe that the only modification to this method needed is:

```
#ifndef ...
    device_rect.roundOut(&device_rect);
#endif
```